diktat-analysis.yml

```diff
@@ -277,6 +277,9 @@
 # Checks year in the copyright
 - name: WRONG_COPYRIGHT_YEAR
   enabled: true
+# Checks copyrights year in configuration
+- name: WRONG_COPYRIGHT_YEAR_CONFIGURATION
```

Comment:
I still think it's not a very good idea to mix config validation with code style warnings. This ensures that the issue gets user's attention, yes, but OTOH you can disable config validation from the config itself and you have one more thing to keep in mind. Let's log a warning in this case, we already have logic that checks copyright year and it will emit a separate warning disregarding value from config.